### PR TITLE
feat: Added some more helpful plugins to make your life better

### PR DESCRIPTION
**Pull request by benjdlambert**
That's right. not 1, but 1 plugin and a quick fix for the `user-settings` plugin has been added here!

You get the benefit of the new plugin `GraphiQL`, and the added extra that the `user-settings` plugin is now configured correctly!

🌈

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2805,6 +2821,16 @@
   version "3.21.0"
   resolved "https://registry.yarnpkg.com/@lerna/run/-/run-3.21.0.tgz#2a35ec84979e4d6e42474fe148d32e5de1cac891"
   integrity sha512-fJF68rT3veh+hkToFsBmUJ9MHc9yGXA7LSDvhziAojzOb0AI/jBDp6cEcDQyJ7dbnplba2Lj02IH61QUf9oW0Q==
+  dependencies:
```
Comment (Rugvip) [inline yarn.lock:2824]: dafuq is up with this O_O, well, hope that fixes the broken build :D

File: yarn.lock [hunk 2]
```diff
@@ -9130,6 +9156,11 @@ graphql-language-service-utils@^2.4.3:
   dependencies:
     graphql-language-service-types "^1.6.3"
 
+graphql@15.3.0:
+  version "15.3.0"
+  resolved "https://registry.yarnpkg.com/graphql/-/graphql-15.3.0.tgz#3ad2b0caab0d110e3be4a5a9b2aa281e362b5278"
+  integrity sha512-GTCJtzJmkFLWRfFJuoo9RWWa/FfamUHgiFosxi/X1Ani4AVWbeyBenZTNX6dM+7WSbbFfTo/25eh0LLkwHMw2w==
```
Comment (Rugvip) [inline yarn.lock:9162]: uh, no deps listed here either. Iirc this is yarn version trouble, perhaps set a policy in this repo too to use a fixed version?

**Review by Rugvip — APPROVED** [review]
:shipit: 🚢  📦

File: yarn.lock (hunk 2 above)
Comment (benjdlambert) [reply yarn.lock:9162]: hmm interesting

**Pull request merged**